challenges/objects-arrays.js

```diff
@@ -100,16 +100,18 @@ console.log(contactInfo)
 
 
 /* Request 3: Find out how many universities have the string "Uni" included in their name. Create a new array called uni that contains them all. Log the result. */
-const uni = [];
+// const uni = [];
+let uni = []
 
-for (let i = 0; i < universities.length; i++) {
-  if (universities[i].includes("Uni")) {
+ for (let i = 0; i < universities.length; i++) {
+   if (universities[i].includes("Uni")) {
     uni.push(universities[i]); 
-  } else {
+   } else {
```

Comment:
You don't need `else {}` here, because `if` just sets up a block of code to run if the statement is true. You only need to add `else` if there's a second block of code to run if the statement is false.
https://www.w3schools.com/js/js_if_else.asp